content/en/continuous_integration/intelligent_test_runner/_index.md

```diff
@@ -19,7 +19,7 @@ Intelligent Test Runner works by analyzing your test suite to determine the code
 
 By minimizing the number of tests run per commit, Intelligent Test Runner reduces the frequency of flaky tests disrupting your pipelines. Flaky tests are tests that may pass or fail at random given the same commit. This can be particularly frustrating when the test flaking is unrelated to the code change being tested. After enabling Intelligent Test Runner for your test services, you can limit each commit to its relevant tests to ensure that flaky tests unrelated to your code change don't end up arbitrarily breaking your build.
 
-### Limitations during beta
+### Limitations
 
 There are known limitations in the current implementation of Intelligent Test Runner that can cause it to skip tests that should be run under certain conditions. Specifically, Intelligent Test Runner is not able to detect changes in library dependencies, compiler options, external services or changes to data files in data-driven tests.
```

Comment:
I think there's a link at the bottom for the dashboard that contains "beta" in the URL

Reply:
I meant to say that I think you can remove the "beta" in the URL and everything will still be fine.

Reply:
Yes, but I wanted to make sure that was really working by deploying the change for the dashboard on staging. All fine :)